src/kernel/src/arch/amd64/memory/pagetables/table.rs

```diff
@@ -81,6 +81,16 @@ impl Table {
         }
         1 << (12 + 9 * level)
```

Comment:
Also, some consts here?

Reply:
I have no idea. Probably, this is the x86_64 side of things.